From 259f32b95c6ac1efdae74f2df53eb322ab8c71de Mon Sep 17 00:00:00 2001
From: Kurt Lee <kurt.lee@cypress.com>
Date: Mon, 8 Feb 2021 01:45:30 -0600
Subject: [PATCH 20/20] defconfig: enable CONFIG_TLS_ADD_DL for DPP

DPP feature has dependency on CONFIG_TLS_ADD_DL thus we should enable
it.
---
 hostapd/defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hostapd/defconfig b/hostapd/defconfig
index cd247832b..548feaaa4 100644
--- a/hostapd/defconfig
+++ b/hostapd/defconfig
@@ -285,7 +285,8 @@ CONFIG_DEBUG_FILE=y
 # internal = Internal TLSv1 implementation (experimental)
 # linux = Linux kernel AF_ALG and internal TLSv1 implementation (experimental)
 # none = Empty template
-#CONFIG_TLS=openssl
+CONFIG_TLS=openssl
+CONFIG_TLS_ADD_DL=y
 
 # TLS-based EAP methods require at least TLS v1.0. Newer version of TLS (v1.1)
 # can be enabled to get a stronger construction of messages when block ciphers
-- 
2.17.1

